packages/lab/src/calendar/Calendar.tsx

```diff
@@ -36,11 +36,19 @@ export interface CalendarBaseProps extends ComponentPropsWithoutRef<"div"> {
   /**
    * Locale for date formatting
    */
+  // biome-ignore lint/suspicious/noExplicitAny: locale is date framework dependent
   locale?: any;
+  /**
+   * Number of visible months, maximum 12, defaults to 1
+   */
+  numberOfVisibleMonths?: ResponsiveProp<
```

Comment:
Given this is `ResponsiveProp`, is it worth to show a responsive example on site, on top of the manually configure with the inputs

Reply:
good shout, alot of value here and it also showcases responsiveness